snap/validate.go

```diff
@@ -47,6 +48,85 @@ func ValidateEpoch(epoch string) error {
 	return nil
 }
 
+// ValidateLicense checks if a string is a valid SPDX expression.
+func ValidateLicense(license string) error {
```

Comment:
I'd personally feel better if this were to build an AST of the license expression rather than doing string peeks.